src/setup-beam.js

```diff
@@ -17,20 +17,27 @@ async function main() {
   const elixirSpec = core.getInput('elixir-version', { required: false })
   const gleamSpec = core.getInput('gleam-version', { required: false })
   const rebar3Spec = core.getInput('rebar3-version', { required: false })
+  const hexMirrors = core.getMultilineInput('hexpm-mirrors', {
```

Comment:
As proposed, in a different comment, we should:
1. have hexMirrors be repo.hex.pm if the input is present but empty
2. have hexMirrors always contain repo.hex.pm (if not already present) at the end, as the last fallback

Reply:
This doesn't follow mix's `HEX_MIRROR` usage.
This PR is also to allow users to entirely opt out of `repo.hex.pm`, for example on hosted runners. Forcing users to always use `repo.hex.pm` doesn't seem right to me.

Reply:
I'll update my request then: the action shall stop with error if `alternative-hexpm-mirrors` is empty (if this is even possible in itself).